src/frontend/csrs-portal/src/app/components/application-form-stepper/application-form-stepper.component.ts

```diff
@@ -66,7 +66,6 @@ export class ApplicationFormStepperComponent implements OnInit {
       @Inject(AccountService) private accountService,
       @Inject(LookupService) private lookupService,
       @Inject(LoggerService) private logger,
-      @Inject(OidcSecurityService) private oidc,
```

Comment:
there is still an import on line 18

`import { OidcSecurityService } from 'angular-auth-oidc-client';`